synctl/cli.py

```diff
@@ -4636,7 +4646,9 @@ def patch_command_options(self):
         patch_exclusive_group.add_argument(
             '--browser', type=str, choices=["chrome", "firefox"], metavar="<string>", help="browser type, support chrome and firefox")
         patch_exclusive_group.add_argument(
-            '-f', '--from-file', type=str, metavar="<filename>", help="specify a script file to update APIScript or BrowserScript")
+            '-f', '--from-file', type=str, metavar="<filename>", help='load synthetic test payload from file (.json)')
```

Comment:
I think we can remove the `-f/--from-file` from the patch command, it is duplicated with the update command `synctl update -f abcd.json`.

Reply:
that is OK to remove `-f/--from-file` from the patch command,